exotic.py

```diff
@@ -572,7 +572,7 @@ def check_file_extensions(directory, filename):
                 # Loop until we find something
                 for exti in file_extensions:
                     for file in os.listdir(directory):
-                        if file.lower().endswith(exti.lower()):
+                        if file.lower().endswith(exti.lower()) and file[0:2] not in ('ref', 'new'):
```

Comment:
I'd be a little careful here- the reference file you make for the WCS coords might not be in index [0:2] depending on how the user begins the names of their FITS files

Reply:
I hard-coded the reference file to always have the first 3 letters to be 'ref' and the plate solution to be 'new'